src/pages/projects-page/AddProjectForm.tsx

```diff
@@ -34,23 +38,30 @@ export function AddProjectForm(props: Props) {
             </Typography>
           </Grid>
           <Grid container item spacing={2} alignItems="center" xs={12}>
-            <Grid item>
-              <DictInputItem field="projectName">
+            <DictInputItem field="projectName">
```

Comment:
Do not move the submit button inside `DictInputItem`. Reset logic should get the fragment from lens, not context